code/__DEFINES/vampire.dm

```diff
@@ -1,6 +1,7 @@
 #define SUBCLASS_HEMOMANCER /datum/vampire_subclass/hemomancer
 #define SUBCLASS_GARGANTUA /datum/vampire_subclass/gargantua
 #define SUBCLASS_UMBRAE /datum/vampire_subclass/umbrae
+#define SUBCLASS_DANTALION /datum/vampire_subclass/dantalion
 #define SUBCLASS_ANCIENT /datum/vampire_subclass/ancient
```

Comment:
What is your ordering here

Reply:
I wanted ancient at the end because its admin only. other than that, theres no real order other than the order I coded them in initially.